test.cc

```diff
@@ -0,0 +1,14 @@
+#include <time.h>
+#include <stdio.h>
+
+int main() {
+    tzset();
+    time_t rawtime;
+    struct tm timeinfo;
+    
+    time(&rawtime);
+    localtime_r(&rawtime, &timeinfo);
```

Comment:
This file, or its close relatives, have been useful for development
but it is probably not a candidate for mainline.

I suggest  squirreling away local copy of this file should be placed in a safe place for future reference and deleting this from this PR.

At the very least, the return from `localtime_r()` (and probably `time()`) should be checked.`

Reply:
Ups sorry 😂